packages/dashboard/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+/dist
+
+# misc
+.DS_Store
+.env.local
```

Comment:
⚠️ Do you also mind adding `.env` to this? (blocking unless there's a reason to allow it)
```suggestion
.env.production.local
.env
```